waiter/src/waiter/cookie_support.clj

```diff
@@ -78,6 +79,12 @@
               (assoc-in response [:headers "set-cookie"] new-header)))]
     (ru/update-response response add-cookie-into-response)))
 
+(defn add-encoded-cookie
+  "Inserts the provided name-value pair as a Set-Cookie header in the response after encoding the value."
+  [response password name value age-in-seconds http-only?]
+  (let [encoded-value (encode-cookie value password)]
```

Comment:
Are we potentially leaking our password in an easy-to-brute-force manner here? Should we be using a per-token salt or something?

https://github.com/twosigma/waiter/blob/e0ca0902c2ecd763fdd844d4675d799c5de2424b/waiter/src/waiter/cookie_support.clj#L60-L64

https://github.com/ptaoussanis/nippy#encryption-v2

Reply:
I agree that if this is an issue, it was already an issue before refactoring. I'm just wondering if this is something we need to address (not necessarily in this PR).

@sradack, thoughts?

Reply:
As a separate PR, we should explore adding a random value into the map for encrypted cookies.